requirements/pip.txt

```diff
@@ -97,5 +97,8 @@ django-cors-middleware==1.3.1
 # User agent parsing - used for analytics purposes
 user-agents<1.2.0
 
+# Utilities used to upload build media to cloud storage
+django-storages>=1.7,<1.8
```

Comment:
I prefer to pin to an exact version our dependencies so the environment is reproducible. You can use something like to allow weekly updates via our bot, but forcing `<1.8`:

     django-storages==1.7.1  # pyup: <1.8